scripts/install_offline_fs_kbc_files.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Alibaba Cloud
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+script_dir="$(dirname $(readlink -f $0))"
+resource_json_name="aa-offline_fs_kbc-resources.json"
+test_resource_record="${script_dir}/../test_data/offline-fs-kbc/${resource_json_name}"
+target_file_path="/etc/${resource_json_name}"
+
+if [ $1 == "install" ]; then
+    cp "${test_resource_record}" "${target_file_path}"
+
+elif [ $1 == "clean" ]; then
+    rm "${target_file_path}"
+
+else
+    echo "Wrong or missing argument"
```

Comment:
This is an error so I'd consider doing something like:

```suggestion
    echo >&2 "ERROR: Wrong or missing argument: '${1:-}'"
```